### ISSUE TITLE
Visual difference between icons from full package and individual weight package

### ISSUE THREAD

**Issue by sream** [marella/material-symbols#34]
As visible within the node screenshots of buttons there is a difference in default configuration. Both are supposed to be weight 400/rounded.

From individual package:

![localhost_4200_login](https://github.com/marella/material-symbols/assets/7093176/542cbc7c-50cf-4046-a642-45e1f7a40afb)

From full package:

![derifin dericon it_login](https://github.com/marella/material-symbols/assets/7093176/7f603901-1e0a-4028-a175-cb9c58b1abe9)

**Comment by UnseenMadness** [comment]
same issue for us as well

**Comment by MelanieW97** [comment]
No idea if that might help locating the problem but the problem seems to be in the woff2 file. I thought I could patch it myself if the error occured in a stylesheet but the woff2 file is already broken.

Full package with font weight 400:
![image](https://github.com/user-attachments/assets/b10bd03d-bbc0-4071-83a3-7351929e3a1f)


Font weight 400 package:
![image](https://github.com/user-attachments/assets/40c44bae-6e50-4a11-81c3-5ef502446b63)

**Comment by MikeSam** [comment]
> As a suggestion (and I plan to do the same), try using @material-symbols/font-600. While it is not exactly the same, the differences, in my personal opinion, are negligible.

I agree they look pretty much the same, except icons `more_vert`, `more_horiz`. They are much thinner (in 400, 600, even 700 versions), than in the full version. I guess this is because individual packages were exported with static value of Optical Size: 48 and it's not possible to change it.

@marella why do you use "opsz" 48 for individual packages? [Google Font](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:more_vert:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=dots&icon.size=24&icon.color=%23e8eaed&icon.platform=web) site shows 24 by default.

![Image](https://github.com/user-attachments/assets/032e5d7a-c791-4ca7-a2fd-a971d61150c6)

<img width="1016" alt="Image" src="https://github.com/user-attachments/assets/70c6a5ac-17de-4783-a481-72712b0eda9d" />
